### ISSUE TITLE
stop/resume jobs

### ISSUE THREAD

**Issue by c-cube** [nunchaku-inria/logitest#15]
- when starting a job, put its full serialized description into a dotfile, and append each individual result as we go (use cbor+decoders?)
- command to list interrupted jobs
- command to resume an interrupted job (subtracting already done individual tasks)

**Comment by c-cube** [comment]
after #23 : 
- create a database as soon as we start the bench
- use a table for "tasks", use it as a queue for workers (with transactional get-and-remove? see what redis does). Tasks include:
  * [ ] discover the expected status of a file, see #10
  * [ ] run one prover on one file 
  * [ ] run a checker on a proof produced by a prover
- if we kill the benchmark system, it just resumes from the list of tasks that resumes

In a way all the state is in the sqlite db from the start.
